Axios 1.12.0 - typescript build issues

### Describe the bug

After upgrading to latest axios version (1.12.0) from (1.11.0) i get this error:

node_modules/axios/index.d.ts:359:28 - error TS2552: Cannot find name 'RequestInfo'. Did you mean 'RequestInit'?

359     Request?: new (input: (RequestInfo | URL), init?: RequestInit) => Request;
                               ~~~~~~~~~~~

node_modules/axios/index.d.ts:360:29 - error TS2304: Cannot find name 'BodyInit'.

360     Response?: new (body?: (BodyInit | null), init?: ResponseInit) => Response;

### To Reproduce

add library to a node project using typescript and run
tsc

### Code snippet

```js

```

### Expected behavior

Compile with out warning

### Axios Version

1.12.0

### Adapter Version

_No response_

### Browser

None (node)

### Browser Version

N/A

### Node.js Version

v22.14.0

### OS

Linux 6.6.87.2-microsoft-standard-WSL2 #1 SMP PREEMPT_DYNAMIC Thu Jun  5 18:30:46 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux

### Additional Library Versions

```bash

```

### Additional context/Screenshots

```bash

```

--- a/index.d.cts
+++ b/index.d.cts
@@ -424,8 +424,11 @@ declare namespace axios {
     env?: {
       FormData?: new (...args: any[]) => object;
       fetch?: (input: URL | Request | string, init?: RequestInit) => Promise<Response>;
-      Request?: new (input: (RequestInfo | URL), init?: RequestInit) => Request;
-      Response?: new (body?: (BodyInit | null), init?: ResponseInit) => Response;
+      Request?: new (input: URL | Request | string, init?: RequestInit) => Request;
+      Response?: new (
+          body?: ArrayBuffer | ArrayBufferView | Blob | FormData | URLSearchParams | string | null,
+          init?: ResponseInit
+      ) => Response;
     };
     formSerializer?: FormSerializerOptions;
     family?: AddressFamily;

--- a/index.d.ts
+++ b/index.d.ts
@@ -356,8 +356,11 @@ export interface AxiosRequestConfig<D = any> {
   env?: {
     FormData?: new (...args: any[]) => object;
     fetch?: (input: URL | Request | string, init?: RequestInit) => Promise<Response>;
-    Request?: new (input: (RequestInfo | URL), init?: RequestInit) => Request;
-    Response?: new (body?: (BodyInit | null), init?: ResponseInit) => Response;
+    Request?: new (input: URL | Request | string, init?: RequestInit) => Request;
+    Response?: new (
+        body?: ArrayBuffer | ArrayBufferView | Blob | FormData | URLSearchParams | string | null,
+        init?: ResponseInit
+    ) => Response;
   };
   formSerializer?: FormSerializerOptions;
   family?: AddressFamily;

